lib/path.js

```diff
@@ -180,6 +180,25 @@ function glob(path, pattern, windows) {
   });
 }
 
+// Regular expressions to identify special device names in Windows.
+// Ref: https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea
+// COM to AUX (e.g., COM1, LPT1, NUL, CON, PRN, AUX) are reserved OS device names.
```

Comment:
```suggestion
//
// COM to AUX (e.g., COM1, LPT1, NUL, CON, PRN, AUX) are reserved OS device names.
```

This might help to separate the ideas

Reply:
You're absolutely right. Moreover, the code I recently added is currently checking all regular expressions in the device namespace, which has significantly reduced performance. As you suggested, I will try to improve performance by distinguishing between special devices and reserved devices during the inspection. Thank you.

Reply:
I added comments to differentiate between reserved and non-reserved device names, explaining the differences between regular expressions like `/([\\/])?(COM\d+)$/i` and `/^(PHYSICALDRIVE\d+)$/i`. While I tried to maintain performance while addressing the issues, I couldn't find a way to do so effectively because reserved device names can also appear in paths (e.g., `C:/path/COM1`). If anyone has a better solution that maintains performance while resolving this, I welcome suggestions.